diff --git a/crates/transaction-fuzzer/scripts/coin_factory_gen.py b/crates/transaction-fuzzer/scripts/coin_factory_gen.py
index 7822ee5..a9cc364 100755
--- a/crates/transaction-fuzzer/scripts/coin_factory_gen.py
+++ b/crates/transaction-fuzzer/scripts/coin_factory_gen.py
@@ -12,6 +12,7 @@ factory_file = os.path.join(os.path.dirname(__file__), os.pardir, "data", "coin_
 
 with open(factory_file, 'w') as f:
     f.write("// Copyright (c) Mysten Labs, Inc.\n")
+    f.write("// Modifications Copyright (c) 2024 IOTA Stiftung\n")
     f.write("// SPDX-License-Identifier: Apache-2.0\n")
 
     f.write("module coiner::coin_factory {\n")
diff --git a/crates/x/src/lint.rs b/crates/x/src/lint.rs
index 0e51004..b245cc4 100644
--- a/crates/x/src/lint.rs
+++ b/crates/x/src/lint.rs
@@ -16,6 +16,7 @@ use nexlint_lints::{
 static EXTERNAL_CRATE_DIR: &str = "external-crates/";
 static CREATE_DAPP_TEMPLATE_DIR: &str = "sdk/create-dapp/templates";
 static LICENSE_HEADER: &str = "Copyright (c) Mysten Labs, Inc.\n\
+                               Modifications Copyright (c) 2024 IOTA Stiftung\n\
                                SPDX-License-Identifier: Apache-2.0\n\
                                ";
 #[derive(Debug, Parser)]
diff --git a/dapps/kiosk/src/Root.tsx b/dapps/kiosk/src/Root.tsx
index e79379d..80d7345 100644
--- a/dapps/kiosk/src/Root.tsx
+++ b/dapps/kiosk/src/Root.tsx
@@ -30,7 +30,7 @@ export default function Root() {
 							<Outlet />
 						</div>
 						<div className="mt-6 border-t border-primary text-center py-6">
-							Copyright © Mysten Labs, Inc.
+							Copyright © Mysten Labs, Inc., Modifications Copyright © 2024 IOTA Stiftung
 						</div>
 						<Toaster position="bottom-center" />
 					</KioskClientProvider>
