diff --git a/apps/icons/svgrrc.config.js b/apps/icons/svgrrc.config.js
index deeea46..8092084 100644
--- a/apps/icons/svgrrc.config.js
+++ b/apps/icons/svgrrc.config.js
@@ -46,6 +46,7 @@ module.exports = {
 		// Insert the copyright header, attached to the first node:
 		template[0].leadingComments = [
 			{ type: 'CommentLine', value: ' Copyright (c) Mysten Labs, Inc.' },
+			{ type: 'CommentLine', value: ' Modifications Copyright (c) 2024 IOTA Stiftung' },
 			{ type: 'CommentLine', value: ' SPDX-License-Identifier: Apache-2.0' },
 		];
 
diff --git a/apps/wallet/prependLicense.mjs b/apps/wallet/prependLicense.mjs
index a40ed05..921e9aa 100644
--- a/apps/wallet/prependLicense.mjs
+++ b/apps/wallet/prependLicense.mjs
@@ -3,7 +3,7 @@
 
 import { readFile, writeFile } from 'fs/promises';
 
-const LICENSE = '// Copyright (c) Mysten Labs, Inc.\n// SPDX-License-Identifier: Apache-2.0\n\n';
+const LICENSE = '// Copyright (c) Mysten Labs, Inc.\n// Modifications Copyright (c) 2024 IOTA Stiftung\n// SPDX-License-Identifier: Apache-2.0\n\n';
 
 async function prependLicense(filename) {
 	const content = await readFile(filename, 'utf8');
